Problem set 2

```diff
@@ -0,0 +1,194 @@
+# 1) read the data, make sure columns are read in the right format, i.e. character, double, etc.
+
+#first column is not the title, check if all columns have the right format
+> library(readr)
+> adult <- read_csv("C:/Users/David/Downloads/adult.txt", 
+                    +     col_names = FALSE)
+Parsed with column specification:
+  cols(
+    X1 = col_double(),
+    X2 = col_character(),
+    X3 = col_double(),
+    X4 = col_character(),
+    X5 = col_double(),
+    X6 = col_character(),
+    X7 = col_character(),
+    X8 = col_character(),
+    X9 = col_character(),
+    X10 = col_character(),
+    X11 = col_double(),
+    X12 = col_double(),
+    X13 = col_double(),
+    X14 = col_character(),
+    X15 = col_character()
+  )
+> View(adult)
+
+# rename columns 
+
+colnames(adult) <- c("Age", "Workclass", "fnlwgt", "Education", "Education_num", "Marital_status", "Occupation", "Relationship", "Race", "Sex", "Gain", "Loss", "Hours_per_week", "Native_country","Salary")
+colnames(adult)
+nrow(adult)
+
+# 2) How many men and women are represented in the dataset? 
+
+only_male <- filter(adult, Sex=="Male")
+only_male
+nrow(only_male)
+# 21790 male in data set 
+
+only_female <- filter(adult, Sex=="Female")
+nrow(only_female)
+# 10771 female in data set 
+
+# final check if there is any NA
+nrow(only_male) + nrow(only_female) - nrow(adult)
+
+#  I have all male and female, the difference is 0
+
+# 3) What is the average age of women
+mean(only_female$Age)
+# just compute mean of Age in female data set
+# Average age of women is circa 37 years
+
+# 4) What are the mean and standard deviation of age for those who earn more than 50K per year
+# and those who earn less than 50K per year
+
+# more than 50K
+
+high_salary <- filter(adult, Salary==">50K")
+mean(high_salary$Age) # mean age is 44
+sd(high_salary$Age) # standart deviation is 10.5
+ 
+
+# less than 50K
+
+low_salary <- filter(adult, Salary=="<=50K")
+mean(low_salary$Age) # mean age is 37
+sd(low_salary$Age) # standart deviation is 14
+
+# 5) Is it true that people who earn more than 50K have at least high school education?
+a <- select(adult, Salary, Education)
+rich_people <- filter(a, Salary==">50K")
+rich_people
+educ_level <- c("Bachelors", "Prof-school", "Assoc-acdm", "Assoc-voc", "Masters or Doctorate")
+educ_level
+final <- str_detect(rich_people$Education, paste(educ_level, collapse = '|'))
+final
+final_df <- data.frame(final)
+head(final_df)
+sum(final_df$final == "TRUE")
+sum(final_df$final == "FALSE")
+
```

Comment:
I don't see the answer to the question, 
for forcing me to guess the answer
-2 points